PR: https://github.com/odoo/odoo/pull/71816

From: 24fc95090a4662ca181e28609e0dcd75e1a3dd2d
From: Ivan Yelizariev
Date: 2021-06-08 16:10:38

Structural Changes: 3
Total Changes: 2

[REM] calendar: delete auto_join=False

it was made for old implementation [1] which is changed since Odoo 14.0 [2]

[1] https://github.com/odoo/odoo/commit/0c07ff7f41a7961985b0f0ef6bd7941b83507822
[2] https://github.com/odoo/odoo/commit/a27afdb5434166c3ea48c18ccfba9e8245d18e62

closes odoo/odoo#71816

Signed-off-by: Ivan Yelizariev // IEL <yelizariev@users.noreply.github.com>

================================= pseudo patch: =================================

--- a/addons/calendar/models/calendar_event.py
+++ b/addons/calendar/models/calendar_event.py
@@ -153,8 +153,6 @@ class Meeting(models.Model):
         'Document Model Name', related='res_model_id.model', readonly=True, store=True)
     # messaging
     activity_ids = fields.One2many('mail.activity', 'calendar_event_id', string='Activities')
-    #redifine message_ids to remove autojoin to avoid search to crash in get_recurrent_ids
-    message_ids = fields.One2many(auto_join=False)
     # attendees
     attendee_ids = fields.One2many(
         'calendar.attendee', 'event_id', 'Participant')
